samples/typescript/11.react-video-js/README.md

```diff
@@ -0,0 +1,92 @@
+# TypeScript: React Video.js Sample
+
+This repository contains a simple app that enables all connected clients to watch videos together using the `live-share-react` package and Video.js.
+
+We have found this structure to be very useful in composing advanced applications with Live Share using Functional React components, but you can compose this differently for your app.
+
+## Getting started
+
+After cloning the repository, you must first set up the npm workspace from the root of the project. Then, run the following commands from the command line:
+
+```bash
+npm install
+npm run build:packages # Build Live Share packages
+cd samples/t*/11*
+```
+
+_Note:_ Do not run `npm start` before running `npm run build:packages` from the root of the project, unless you first move the sample out of this npm workspace. When using our samples, you are testing the packages using symlinks, and not the Live Share SDK versions published to npm.
```

Comment:
installing the dependencies is enough for live share to be built, we can probably remove this from the other samples if it exists there too

Reply:
Good catch, fixed :)